bitnami/flux/templates/helm-controller/networkpolicy.yaml

```diff
@@ -18,14 +18,14 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
+  policyTypes:
+    - Ingress
+    - Egress
   {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.helmController.podLabels .Values.commonLabels ) "context" . ) }}
   podSelector:
     matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 6 }}
       app.kubernetes.io/part-of: flux
       app.kubernetes.io/component: helm-controller
-  {{- if .Values.helmController.networkPolicy.extraEgress }}
-  egress: {{- include "common.tplvalues.render" ( dict "value" .Values.helmController.networkPolicy.extraEgress "context" $ ) | nindent 4 }}
-  {{- end }}
```

Comment:
Why do we want to remove this? It seems like the param in `values.yaml` has not been removed

Reply:
This section is duplicated, that's why we're removing it.